docs/reference/glossary-of-terms.md

```diff
@@ -127,12 +136,18 @@ The systems or platform where various types of ontologies are stored from differ
 #### Ontology Repository 
 A curated collection of ontologies.
 
+### OTTR
+[Reasonable Ontology Templates](https://ottr.xyz/). A system for composable ontology templates and documentation.
+
 #### OWL
 An ontology language that uses constructs from [Description Logic](#Description-Logic). OWL is not itself an ontology format, it can be serialized through different formats such as [Functional Syntax](#Functional-Syntax), and it can be mapped to :[RDF](#RDF) and serialized via an RDF format.
```

Comment:
Consider adding what this stands for (i.e., Web Ontology Language with the letters reorganised for easier articulation).